Lib/test/test_asyncio/test_base_events.py

```diff
@@ -1316,7 +1316,8 @@ def mock_make_ssl_transport(sock, protocol, sslcontext, waiter,
         self.loop._make_ssl_transport.assert_called_with(
             ANY, ANY, ANY, ANY,
             server_side=False,
-            server_hostname='python.org')
+            server_hostname='python.org',
+            ssl_handshake_timeout=ANY)
```

Comment:
Using `ANY` kind of defeats the purpose of the test.  Do this:

```python
handshake_timeout = object()
```

and then pass it instead of `ANY` and assert calls with that `handshake_timeout` object.